libs/features/georoutingv2/georoutingv2.css

```diff
@@ -89,7 +89,11 @@
 
 .dialog-modal.locale-modal-v2 .tabs div[role="tablist"] .tab-list-container {
   padding: initial;
-  margin: initial;
+  margin: 2px;
+}
+
+.dialog-modal.locale-modal-v2 .tabs.quiet div[role="tablist"] button {
+  border-radius: 0!important;
```

Comment:
I don't see a need to use the !important. These usually surface up problems else were and for the most part should be avoided.

Removing the !important gives the same results.

Also, you have it scoped to the `quiet` tabs variant. In the event that ever changes this will not work. 
It would be better to simplify the scope to:

```
.dialog-modal.locale-modal-v2 .tabs button {
    border-radius: 0;
}
```
This still works as expected and will work for all tabs variations in the geoModal. 

![image](https://github.com/adobecom/milo/assets/2539954/aa1861ff-1fb7-401a-965d-0359d584cfd3)